spec/controllers/idv/in_person/ssn_controller_spec.rb

```diff
@@ -3,10 +3,11 @@
 RSpec.describe Idv::InPerson::SsnController do
   include IdvHelper
 
+  let(:pii_from_user) { Idp::Constants::MOCK_IDV_APPLICANT_SAME_ADDRESS_AS_ID_WITH_NO_SSN.dup }
+
   let(:flow_session) do
     { 'document_capture_session_uuid' => 'fd14e181-6fb1-4cdc-92e0-ef66dad0df4e',
-      'pii_from_user' => Idp::Constants::MOCK_IDV_APPLICANT_SAME_ADDRESS_AS_ID_WITH_NO_SSN.dup,
-      :threatmetrix_session_id => 'c90ae7a5-6629-4e77-b97c-f1987c2df7d0',
```

Comment:
do we not need tests for threatmetrix?

Reply:
@svalexander  Good point! I put threatmetrix_session_id back onto flow_session and added two tests.  I can see threatmetric_session_id on flow_session on ssn step for both fsm and non-fsm.

See commit 073e2f29b6c6b5909fa49ca4f7f8aaeadde2c8c4